srcpkgs/runit-iptables/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'runit-iptables'
+pkgname=runit-iptables
+version=20180616
+revision=1
+noarch=yes
+depends="runit-void iptables"
+short_desc="Restore iptables rules on boot"
+maintainer="Nicolas Porcel <nicolasporcel06@gmail.com"
+license="Public Domain"
+homepage="http://www.voidlinux.eu/"
```

Comment:
Why not http://www.voidlinux.org/

Reply:
I copied the template of `runit-kdump`. Also, I just did a grep on all templates of the other packages and none of them is using <http://www.voidlinux.org>.

If the new policy is to use <http://www.voidlinux.org> I can change it.

Reply:
Yes, that is the new domain going forward.

Reply:
OK I will update it.